### ISSUE TITLE
Add onfailure handler to pulseaudio service config

### ISSUE THREAD

**Issue by ppp0** [cargomedia/puppet-packages#1354]
To make sure that `lightdm` restarts after a crash of `pulseaudio`

**Comment by ppp0** [comment]
depends on #1355 

Idea:
- PA service will have `After=lightdm.service`
- PA service will have `ExecStartPost=/bin/systemd try-restart lightdm`

**Comment by kris-lab** [comment]
Closing in favor of https://github.com/cargomedia/puppet-packages/issues/1359
